pro/src/pages/OfferIndividualWizard/Confirmation/__specs__/Confirmation.spec.tsx

```diff
@@ -0,0 +1,259 @@
+      accessibility: {
+        [AccessiblityEnum.AUDIO]: false,
+        [AccessiblityEnum.MENTAL]: false,
+        [AccessiblityEnum.MOTOR]: false,
+        [AccessiblityEnum.VISUAL]: false,
+        [AccessiblityEnum.NONE]: true,
+      },
+      managingOffererId: 'OFID',
+      withdrawalDetails: '',
+    }
+
+    offer = {
+      id: 'AA',
+      nonHumanizedId: 12,
+      author: 'Offer author',
+      bookingEmail: 'booking@email.com',
+      description: 'Offer description',
+      durationMinutes: 140,
+      isbn: '',
+      isActive: true,
+      isDuo: false,
+      isEducational: false,
+      isEvent: true,
+      isDigital: false,
+      accessibility: {
+        [AccessiblityEnum.AUDIO]: true,
+        [AccessiblityEnum.MENTAL]: true,
+        [AccessiblityEnum.MOTOR]: true,
+        [AccessiblityEnum.VISUAL]: true,
+        [AccessiblityEnum.NONE]: false,
+      },
+      isNational: false,
+      name: 'Offer name',
+      musicSubType: '',
+      musicType: '',
+      offererId: 'OFID',
+      offererName: '',
+      performer: 'Offer performer',
+      showSubType: '',
+      showType: '',
+      stageDirector: 'Offer stageDirector',
+      speaker: 'Offer speaker',
+      subcategoryId: 'SCID physical',
+      image: undefined,
+      url: 'http://offer.example.com',
+      externalTicketOfficeUrl: 'http://external.example.com',
+      venueId: 'VID physical',
+      venue: {
+        id: 'VID physical',
+        name: 'Venue name',
+        publicName: 'Venue publicName',
+        isVirtual: false,
+        address: '15 rue de la corniche',
+        postalCode: '75001',
+        city: 'Paris',
+        offerer: {
+          id: 'OFID',
+          name: 'Offerer name',
+        },
+        departmentCode: '75',
+        accessibility: {
+          [AccessiblityEnum.AUDIO]: false,
+          [AccessiblityEnum.MENTAL]: false,
+          [AccessiblityEnum.MOTOR]: false,
+          [AccessiblityEnum.VISUAL]: false,
+          [AccessiblityEnum.NONE]: true,
+        },
+      },
+      visa: '',
+      withdrawalDetails: 'Offer withdrawalDetails',
+      withdrawalDelay: 140,
+      withdrawalType: WithdrawalTypeEnum.ON_SITE,
+      stocks: [],
+      lastProviderName: null,
+      lastProvider: null,
+      status: OfferStatus.ACTIVE,
+    }
+
+    contextOverride = {
+      offerId: offer.id,
+      offer: offer,
+      venueList: [
+        venue,
+        {
+          id: 'VID virtual',
+          name: 'Lieu online BB',
+          managingOffererId: 'OFID',
+          isVirtual: true,
+          withdrawalDetails: '',
+          accessibility: {
+            visual: false,
+            mental: false,
+            audio: false,
+            motor: false,
+            none: true,
+          },
+        },
+      ],
+      offererNames: [{ id: 'OFID', name: 'Offerer name' }],
+      categories,
+      subCategories,
+    }
+    jest
+      .spyOn(api, 'getOffer')
+      .mockResolvedValue({} as GetIndividualOfferResponseModel)
```

Comment:
Afin de rendre ca plus lisible, j'utiliserai bien des `as MonInterface` ne contenant que ce qui est effectivement utilisé.

```suggestion
    offer = {
      nonHumanizedId: 12,
      venueId: 'VID physical',
      venue: {
        offerer: {
          id: 'OFID',
          name: 'Offerer name',
        },
      },
      status: OfferStatus.ACTIVE,
    } as IOfferIndividual

    contextOverride = {
      offer: offer,
    }
    jest
      .spyOn(api, 'getOffer')
      .mockResolvedValue({} as GetIndividualOfferResponseModel)
```